### PR TITLE
Transparent logo develop

### PR DESCRIPTION
**Pull request by hritikaggarwal31**

### CODE REVIEW

**Comment by sourobrata** [comment]
@hritikaggarwal31 Please resolve the conflicts

**Comment by hritikaggarwal31** [comment]
[Transparent logos for PNG QR codes](https://trello.com/c/P9z36yLe/1330-transparent-logos-for-png-qr-codes)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "mobstac-awesome-qr",
-    "version": "2.1.10",
+    "version": "2.1.8-beta.27",
```
Comment (sourobrata) [inline package.json:3]: @hritikaggarwal31 The patch number should not be less than previous versions.

File: src/tests/QR.test.ts [hunk 2]
```diff
@@ -1420,6 +1462,37 @@ describe('Output QR code tests', () => {
...
+            
+            
+                
+            const dataUrl = qrCode.canvas.toDataURL('image/png');
+            const matches: any = dataUrl.match(
+                    /^data:([A-Za-z-+\/]+);base64,(.+)$/
+                    ),
+                response: any  ={};
+            response.type = matches[1];
+            response.data = Buffer.from(matches[2], "base64");
```
Comment (sourobrata) [inline src/tests/QR.test.ts:1478]: @hritikaggarwal31 Please use prepareImageBuffer()

**Comment by sourobrata** [comment]
@hritikaggarwal31 Please add newly generated test QR code files to compare_files.sh

File: src/tests/QR.test.ts (hunk 2 above)
Comment (hritikaggarwal31) [reply src/tests/QR.test.ts]: Fixed.

**Review by sourobrata — APPROVED** [review]

**Pull request merged**